.gitignore

```diff
@@ -287,6 +287,9 @@ FakesAssemblies/
 .ntvs_analysis.dat
 node_modules/
 
+# Visual Studio solution file
+*.sln
```

Comment:
Why do you want to ignore changes the the solution file?

Reply:
I see there were to SLN files.  Why not keep one of them?

Reply:
I removed it because it's not needed to run the project. but I've added it back